### ISSUE TITLE
Indentation problem with line including word 'end'

### ISSUE THREAD

**Issue by gasparch** [elixir-lang/vim-elixir#267]
When selecting this text and pressing `=` it gets indented wrongly

```elixir
  fun1 = fn
    (:foo) ->
      :bar
      :end
  end

  fun2 = fn
    (:foo) ->
      :bar
      'end'
  end

  fun3 = fn
    (:foo) ->
      :bar
      :send
  end
```

## Actual

```elixir
  fun1 = fn
    (:foo) ->
      :bar
    :end
  end

  fun2 = fn
    (:foo) ->
      :bar
    'end'
end

fun3 = fn
  (:foo) ->
    :bar
  :send
end
```

Pressing `<Enter>` after `:bar` puts cursor in correct position. It is just when whole text selected and `=` is pressed problem occurs.

**Comment by jbodah** [comment]
Thanks for these reports @gasparch, very helpful

**Comment by gasparch** [comment]
Welcome :) 
I'm putting together IDE based on this and many other plugins, so I test them intensively + day to day work :)) :P 

Check it out here - https://github.com/gasparch/vim-ide-elixir 

installation + feedback are highly welcome :)

**Comment by jbodah** [comment]
Fixed in 3621b9253aabb629b3a7280b241726444bef2a66
